### ISSUE TITLE
Push CircleCI messages to multiple Slack organizations/channels

### ISSUE THREAD

**Issue by thinkh** [phovea/generator-phovea#210]
**Current Status**

We are using the [CircleCI integration for Slack](https://slack.com/apps/A0F7VRE7N-circleci) and add them to specific channels in our Slack organizations. 

Following [the instructions](https://circleci.com/blog/slack-integration/), we get a webhook URL in the configuration step of the integration that must be added to the CircleCI project configuration. 

The problem that only one webhook can be registered per CircleCI project. Hence, CircleCi can only push the status to a single Slack channel.

**Solution**

It should be possible to push status messages to multiple channels and organziations. That might be possible if we use and integrate the Slack Orb in the CircleCI configuration. However, further research and test might be required.

**Links**

* [Enabling Custom Slack Notifications in CircleCI Jobs](https://circleci.com/docs/2.0/configuration-cookbook/#header5)
* [Slack Orb](https://github.com/CircleCI-Public/slack-orb) for CircleCI

**Comment by keckelt** [comment]
Yes, with the Orb you can also overwrite the webhook url. So you should be able to add multiple steps for multiple channels and setindividual channel's webhook url there.
